src/timidity/timidity.h

```diff
@@ -19,7 +19,11 @@ typedef Sint32 final_volume_t;
 #define VIBRATO_SAMPLE_INCREMENTS 32
 
 /* Maximum polyphony. */
-#define MAX_VOICES	48
+/* #define MAX_VOICES	48 */
+#define MAX_VOICES	256
+#define MAXCHAN	16
+/* #define MAXCHAN	64 */
+#define MAXBANK	128
```

Comment:
same ..

Reply:
This one hasn't changed from the previous value.

Reply:
my bad